scripts/apps/archive/views/preview.html

```diff
@@ -151,6 +151,8 @@
 
             <div class="text dateline" order="{{editor['dateline'].order}}" ng-show="selected.preview.dateline" sd-html-preview="selected.preview.dateline.text"></div>
 
+            <div class="text body-text" order="{{editor['event_description'].order}}" ng-show="selected.preview.description_text" sd-html-preview="selected.preview.description_text" item="selected.preview"></div>
```

Comment:
not sure about that `event_description` for order? why is that?

Reply:
Because IMO if we use `event_description` for order, the `item.description` it looks like this.
![SDESK-6618 (1)](https://user-images.githubusercontent.com/73937490/196623156-097007db-b1d9-49c6-8048-06bb9702435e.png)

Reply:
I would probably rather do this like for [image description](https://github.com/superdesk/superdesk-client-core/pull/4135/files#diff-32d5353b9ef1fdafaa57a2b7353e61c62ce7a2647a7f616138a74603ff602727R136) so it would be always after the media player